package.json

```diff
@@ -37,11 +37,11 @@
     "pify": "^2.2.0",
     "rimraf": "^2.4.3",
     "run-applescript": "^2.0.0",
-    "xdg-trashdir": "^2.0.0"
+    "xdg-trashdir": "^2.1.0",
+    "path-exists": "^2.0.0"
   },
   "devDependencies": {
     "ava": "*",
-    "path-exists": "^2.0.0",
```

Comment:
Keep it here too if it's needed in the tests. They might break in the future if we remove it from the `dependencies`.